lib/datadog/security/waf.rb

```diff
@@ -0,0 +1,68 @@
+require 'libsqreen'
+require 'datadog/security/assets'
+
+module Datadog
+  module Security
+    module WAF
+      class Args < Hash; end
+
+      class Rules < Hash; end
+
+      module_function
+
+      # TODO: logger
+      def logger
+        @logger ||= ::Logger.new(STDOUT)
+        @logger.level = ::Logger::DEBUG
+        #@logger.level = ::Logger::WARN
+        @logger.debug { 'waf logger enabled' }
+        @logger
+      end
+
+      def rules
+        @rules ||= JSON.dump(JSON.parse(Assets.waf_rules))
+      end
+
+      def name
+        rule_name = 'waf_rules'
+        @name ||= format('%s_%s', SecureRandom.uuid, rule_name)
+      end
```

Comment:
I don't quite understand this naming -- what's the purpose of having "name" include a random UUID in this context?